addons/binding/org.openhab.binding.autelis/src/main/java/org/openhab/binding/autelis/handler/AutelisHandler.java

```diff
@@ -341,22 +367,66 @@ protected void execute() {
          * This xmlDoc will now contain the three XML documents we retrieved
          * wrapped in response tags for easier querying in XPath.
          */
+        HashMap<String, String> pumps = new HashMap<>();
         String xmlDoc = sb.toString();
         for (Channel channel : getThing().getChannels()) {
-            String key = channel.getUID().getId().replace('-', '/');
+            String key = channel.getUID().getId().replaceFirst("-", "/");
```

Comment:
Is it really desired to only replace the first occurrence?

Reply:
Yes, its how autelis formats their variables.